### ISSUE TITLE
Ubuntu install error

### ISSUE THREAD

**Issue by lemonadezZ** [MapServer/MapServer#5539]
```
cmake -DCMAKE_INSTALL_PREFIX=/opt  -DWITH_PROJ=1  -DWITH_FRIBIDI=0  -DWITH_HARFBUZZ=0 -DWITH_FCGI=0 -DWITH_GEOS=0 -DWITH_POSTGIS=0 -DWITH_GDAL=0 -DWITH_OGR=0 -DWITH_WFS=0 -DWITH_WCS=0  -DWITH_CLIENT_WMS=0 -DWITH_GIF=0 -DCMAKE_PREFIX_PATH=/opt ..
```
error
```
[  1%] Building C object CMakeFiles/mapserver.dir/mapio.c.o
/root/mapserver-rel-7-0-7/mapio.c: In function ‘msIO_getAndStripStdoutBufferMimeHeaders’:
/root/mapserver-rel-7-0-7/mapio.c:790:10: error: ‘TRUE’ undeclared (first use in this function)
   while( TRUE ) {
          ^
/root/mapserver-rel-7-0-7/mapio.c:790:10: note: each undeclared identifier is reported only once for each function it appears in
CMakeFiles/mapserver.dir/build.make:854: recipe for target 'CMakeFiles/mapserver.dir/mapio.c.o' failed
make[2]: *** [CMakeFiles/mapserver.dir/mapio.c.o] Error 1
CMakeFiles/Makefile2:215: recipe for target 'CMakeFiles/mapserver.dir/all' failed
make[1]: *** [CMakeFiles/mapserver.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
```
Ubuntu version
```
cat /proc/version
Linux version 4.4.0-75-generic (buildd@lgw01-21) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.4) ) #96-Ubuntu SMP Thu Apr 20 09:56:33 UTC 2017
```

**Comment by bjoernboldt** [comment]
"TRUE" is defined in /usr/include/gdal/cpl_progress.h:33 included  from /usr/include/gdal/ogr_api.h:42
included from mapserver.h:146, but only if USE_OGR is defined (see mapserver.h:144).
So try to use -DWITH_OGR=1.

**Comment by szekerest** [comment]
Another issue here is that

_#define inline __inline_

is coming from cpl_config.h

**Comment by aimdev** [comment]
This issue is repeatable on debian jessie.
To resolve this the libgdal-dev library is required, which raises the issue that the documentation for
Version 7.0.7  (http://mapserver.org/installation/unix.html) states that GDAL is highly recommended, when in fact it is mandatory to do a successful compile.
Recommend documentation is updated to avoid confusion.

**Comment by sdlime** [comment]
I create a new ticket in docs to address the install documentation update. Can the ticket be closed? --Steve

**Comment by aimdev** [comment]
Hi

Yes, can be closed
Thanks

Aimee

**Comment by jratike80** [comment]
Closing now.
